js/lib/beautify-html.js

```diff
@@ -396,16 +401,21 @@
                     if (content.length && content[content.length - 1] !== '=' && input_char !== '>' && space) {
                         //no space after = or before >
                         var wrapped = this.space_or_wrap(content);
-                        var indentAttrs = wrapped && input_char !== '/' && wrap_attributes !== 'force';
+                        var indentAttrs = wrapped && input_char !== '/' && !this.startWidth(wrap_attributes, 'force');
```

Comment:
You don't want to be doing this search every time. Do it once early on and save it in var.